custom_components/qss/manifest.json

```diff
@@ -6,6 +6,6 @@
   "documentation": "https://github.com/CM000n/QSS",
   "iot_class": "local_push",
   "issue_tracker": "https://github.com/CM000n/QSS/issues",
-  "requirements": ["questdb>=1.2,<2.0", "tenacity>=8.0"],
+  "requirements": ["questdb==2.0.3", "tenacity>=8.0"],
```

Comment:
shouldn't we at least support all patch and minor version jumps instead of pinning a specific version?

Reply:
@CM000n no, why? when you pin the dep … you do a release on which you know that it works. someone at questdb releases a breaking version, which results in issues opened here.

Reply:
If they take semantic versioning seriously, minor patch releases should never break the API. With a pin, I see a greater risk that users will then remain on a lower version than currently available and not notice and use security-relevant patches.
Unfortunately, I don't have the time or inclination to publish a Qss release for every questdb patch release.

Reply:
> shouldn't we at least support all patch and minor version jumps instead of pinning a specific version?

only 2.0.3 is actually useable and buildable for real :)
we can make it ">=2.0.3", hopping they will not break it again, but this is the most we can do

Reply:
> If they take semantic versioning seriously, minor patch releases should never break the API. With a pin, I see a greater risk that users will then remain on a lower version than currently available and not notice and use security-relevant patches. Unfortunately, I don't have the time or inclination to publish a Qss release for every questdb patch release.

while 2.0.x should be compatible, but they have build system broken due to bugs. Asking for ">=2.0" can result in potential generation of problems.